app/export.go

```diff
@@ -62,6 +65,31 @@ var exportablePreferences = map[ComparablePreference]string{{
 }: "EmailInterval",
 }
 
+func (a *App) GetId() string {
```

Comment:
```suggestion
func (a *App) GetID() string {
```
change this everywhere